src/Optimization.jl

```diff
@@ -121,8 +121,12 @@ will be included in the `Options` constructor of Optim.jl.
 Convergence status is stored in the field `ret.converged` of the return value
 `ret`. Additional optimization statistics are stored in the field `ret.data`.
 """
-function minimize_energy!(sys::System{N}; maxiters=1000, subiters=10, method=Optim.ConjugateGradient(),
-                          kwargs...) where N
+function minimize_energy!(sys::System{N}; maxiters=1000, method=Optim.ConjugateGradient(),
+                          subiters=10, δ=1e-8, kwargs...) where N
+    # Perturbation of sufficient magnitude to "almost surely" push away from an
```

Comment:
"Almost surely" is based on the assumption that the unstable minimum "has measure zero" (i.e., in principle any perturbation should work)?

Reply:
Correct